tests/rules/test_gcloud_cli.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+from thefuck.rules.gcloud_cli import match, get_new_command
+from thefuck.types import Command
+
+
+no_suggestions = '''\
+ERROR: (gcloud) Command name argument expected.
+'''
+
+
+misspelled_command = '''\
+ERROR: (gcloud) Invalid choice: 'comute'.
+Usage: gcloud [optional flags] <group | command>
+  group may be           access-context-manager | ai-platform | alpha | app |
+                         asset | auth | beta | bigtable | builds | components |
+                         composer | compute | config | container | dataflow |
+                         dataproc | datastore | debug | deployment-manager |
+                         dns | domains | endpoints | filestore | firebase |
+                         functions | iam | iot | kms | logging | ml |
+                         ml-engine | organizations | projects | pubsub | redis |
+                         resource-manager | scheduler | services | source |
+                         spanner | sql | tasks | topic
+  command may be         docker | feedback | help | info | init | version
+
+For detailed information on this command and its flags, run:
+  gcloud --help
+'''
+
+
+misspelled_subcommand = '''\
+ERROR: (gcloud.compute) Invalid choice: 'instance'.
+Maybe you meant:
+  gcloud compute instance-groups
+  gcloud compute instance-templates
+  gcloud compute instances
+  gcloud compute target-instances
+
+To search the help text of gcloud commands, run:
+  gcloud help -- SEARCH_TERMS
+'''
+
+
+
+
+@pytest.mark.parametrize('command', [
```

Comment:
Quoting build [#1932.2](https://travis-ci.org/github/nvbn/thefuck/jobs/680589249#L460) on TravisCI:
> ./tests/rules/test_gcloud_cli.py:46:1: E303 too many blank lines (4)

Reply:
Thanks didn't see that error. I have removed duplicate blank lines

Reply:
I'll try and get it green best I can, thanks for the input

Reply:
Having a little trouble figuring out why the assertions in pytest aren't returning the correct value, I haven't used pytest before

Reply:
You got it all green! Awesome! pytest is great and I hope you're learning a good deal of it with this PR 🙂 When in doubt, don't hesitate to ask. Also, tests for other rules may serve as a good reference.